### ISSUE TITLE
Bootstrap install fails

### ISSUE THREAD

**Issue by ivanbacher** [jspm/registry#1073]
```
jspm install bootstrap

Updating registry cache...
     Looking up github:twbs/bootstrap
     Downloading github:twbs/bootstrap@4.0.0

warn Error on processPackageConfig
     Package.json dependency tether set to github:HubSpot/tether@^1.1.1, which is not a valid dependency format for npm.
     It's advisable to publish jspm-style packages to GitHub or another registry so conventions are clear.
     To skip npm compatibility install with jspm install github:twbs/bootstrap@4.0.0 -o "{jspmPackage: true}".

warn Error processing package config for github:twbs/bootstrap.
(node:13807) [DEP0013] DeprecationWarning: Calling an asynchronous function without callback is deprecated.

err  Error processing package config for github:twbs/bootstrap.

warn Installation changes not saved.
```

**Comment by ivanbacher** [comment]
installing bootstrap using npm: `jspm install bootstrap=npm:bootstrap` seems to work without producing any errors

**Comment by Lasantha77** [comment]
Im getting following error  when I install Bootstrap with bellow command 
jspm install aurelia-framework aurelia-bootstrapper bootstrap

Error processing package config for github:twbs/bootstrap.
(node:17716) [DEP0013] DeprecationWarning: Calling an asynchronous function without callback is deprecated.

Any Idea Please?

**Comment by ivanbacher** [comment]
@Lasantha77 Try installing bootstrap using this command: `jspm install bootstrap=npm:bootstrap`

**Comment by bfil** [comment]
Installing using `npm:bootstrap` works but I think the registry override is messing up the dependencies configuration because the latest version of bootstrap uses `popper.js` instead of `tether`:
https://github.com/twbs/bootstrap/blob/v4-dev/package.json#L216

This is making SystemJS load bootstrap before popper causing the error `Bootstrap tooltips require Popper.js`.

Anyone else experiencing this?

**Comment by greenlaw** [comment]
@bfil Yes, I'm also encountering the `Bootstrap tooltips require Popper.js` error.

Even though I can `import Popper from "popper.js"` without any issues, Bootstrap itself can't seem to import it.
